pkg/diff/diff_test.go

```diff
@@ -222,7 +221,7 @@ func createTableDiff(db *sql.DB) *TableDiff {
 }
 
 func createConn() (*sql.DB, error) {
-	return dbutil.OpenDB(dbutil.GetDBConfigFromEnv("test"))
+	return dbutil.OpenDB(dbutil.GetDBConfigFromEnv(""))
```

Comment:
why not use `dbutil.GetDBConfigFromEnv("test")` here anymore?

Reply:
some mysql instance may not have database `test`